src/pages/BeerPage/Components/BeerMethod/BeerMethod.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import "./style.sass";
+
+function BeerMethod({ method }) {
+  return (
+    <div className="method">
+      <h2>Method</h2>
+
+      <h3>Mash</h3>
+      {method.mash_temp.map((item) => {
```

Comment:
Is better to keep it outside return block

Reply:
Done